From 48e7bda5d92f3ff3eff3c2eadd5f8a4452cb666a Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:48 -0400
Subject: [redhat] kabi: Adding symbol blkdev_issue_discard (block/blk-lib.c)

Message-id: <ab706eeda68aa7b9d0b51e66dcf62183415a7eb1.1410510773.git.poros@redhat.com>
Patchwork-id: 95955
O-Subject: [PATCH RHEL7.1 048/233] kabi: Adding symbol blkdev_issue_discard (block/blk-lib.c)
Bugzilla: 1071097
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Issue a discard request for the sectors in question.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/blkdev_issue_discard
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/blkdev_issue_discard
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/blkdev_issue_discard

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/blkdev_issue_discard b/redhat/kabi/kabi-module/kabi_ppc64/blkdev_issue_discard
new file mode 100644
index 0000000..b59979a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/blkdev_issue_discard
@@ -0,0 +1,2 @@
+#1-
+0x537208c4 blkdev_issue_discard vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/blkdev_issue_discard b/redhat/kabi/kabi-module/kabi_s390x/blkdev_issue_discard
new file mode 100644
index 0000000..06c9aee
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/blkdev_issue_discard
@@ -0,0 +1,2 @@
+#1-
+0xcf4642fd blkdev_issue_discard vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/blkdev_issue_discard b/redhat/kabi/kabi-module/kabi_x86_64/blkdev_issue_discard
new file mode 100644
index 0000000..99f6bf1
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/blkdev_issue_discard
@@ -0,0 +1,2 @@
+#1-
+0x9a0d63fa blkdev_issue_discard vmlinux EXPORT_SYMBOL
-- 
1.7.1